README.md

```diff
@@ -13,7 +13,7 @@ build toolchain is needed). If you wanted to check with `pygatt` then checkout v
 Install dependencies:
 
 ```
-pip3 install asyncio bleak paho-mqtt
+pip3 install -r requirements.txt
```

Comment:
Since requirements.txt has a hard lock on versions and more suitable for docker, lets keep install section as it was before.

Alternative is to relax version lock in requirements.txt

Reply:
IMO version lock is necessary for stability